runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/SkillChallengeClue.java

```diff
@@ -0,0 +1,347 @@
+		new SkillChallengeClue("Score a goal in skullball.", true, any("Ring of Charos", item(ItemID.RING_OF_CHAROS), item(ItemID.RING_OF_CHAROSA))),
+		new SkillChallengeClue("Complete a lap at the Ape Atoll Agility Course.", true, any("Ninja Monkey Greegree", item(ItemID.NINJA_MONKEY_GREEGREE), item(ItemID.NINJA_MONKEY_GREEGREE_4025))),
+		new SkillChallengeClue("Create a super defence potion.", item(ItemID.CADANTINE_POTION_UNF), item(ItemID.WHITE_BERRIES)),
+		new SkillChallengeClue("Steal from a chest in King Lathas' castle in East Ardougne.", item(ItemID.PURE_ESSENCE)),
+		new SkillChallengeClue("Craft a green d'hide body.", xOfItem(ItemID.GREEN_DRAGON_LEATHER, 3), item(ItemID.NEEDLE), item(ItemID.THREAD)),
+		new SkillChallengeClue("String a yew longbow.", item(ItemID.YEW_LONGBOW_U), item(ItemID.BOW_STRING)),
+		new SkillChallengeClue("Kill a Dust Devil.", "slay a dust devil.", true, any("Facemask or Slayer Helmet", item(ItemID.FACEMASK), item(ItemID.SLAYER_HELMET), item(ItemID. SLAYER_HELMET_I), item(ItemID. BLACK_SLAYER_HELMET), item(ItemID. BLACK_SLAYER_HELMET_I), item(ItemID. PURPLE_SLAYER_HELMET), item(ItemID. PURPLE_SLAYER_HELMET_I), item(ItemID. RED_SLAYER_HELMET), item(ItemID. RED_SLAYER_HELMET_I), item(ItemID.GREEN_SLAYER_HELMET), item(ItemID. GREEN_SLAYER_HELMET_I), item(ItemID. TURQUOISE_SLAYER_HELMET), item(ItemID. TURQUOISE_SLAYER_HELMET_I), item(ItemID. HYDRA_SLAYER_HELMET), item(ItemID. HYDRA_SLAYER_HELMET_I))),
+		new SkillChallengeClue("Catch a black warlock.", item(ItemID.BUTTERFLY_JAR), any("Butterfly Net", item(ItemID.BUTTERFLY_NET), item(ItemID.MAGIC_BUTTERFLY_NET))),
+		new SkillChallengeClue("Catch a red chinchompa.", item(ItemID.BOX_TRAP)),
+		new SkillChallengeClue("Mine a piece of mithril ore.", ANY_PICKAXE),
+		new SkillChallengeClue("Smith a mithril 2h sword.", item(ItemID.HAMMER), xOfItem(ItemID.MITHRIL_BAR, 3)),
+		new SkillChallengeClue("Catch a raw shark.", any("Harpoon", item(ItemID.HARPOON), item(ItemID.BARBTAIL_HARPOON), item(ItemID.DRAGON_HARPOON), item(ItemID.INFERNAL_HARPOON), item(ItemID.INFERNAL_HARPOON_UNCHARGED))),
+		new SkillChallengeClue("Chop a yew tree.", ANY_AXE),
+		new SkillChallengeClue("Fix a magical lamp in Dorgesh-Kaan.", item(ItemID.LIGHT_ORB)),
+		new SkillChallengeClue("Burn a yew log.", item(ItemID.YEW_LOGS), item(ItemID.TINDERBOX)),
+		new SkillChallengeClue("Catch and cook a swordfish.", "cook a swordfish.", any("Harpoon", item(ItemID.HARPOON), item(ItemID.BARBTAIL_HARPOON), item(ItemID.DRAGON_HARPOON), item(ItemID.INFERNAL_HARPOON), item(ItemID.INFERNAL_HARPOON_UNCHARGED))),
+		new SkillChallengeClue("Craft multiple cosmic runes from a single essence.", item(ItemID.PURE_ESSENCE)),
+		new SkillChallengeClue("Plant a watermelon seed.", item(ItemID.RAKE), item(ItemID.SEED_DIBBER), xOfItem(ItemID.WATERMELON_SEED, 3)),
```

Comment:
Rakes aren't needed, particularly if you have auto-weed enabled, or are replacing a freshly-harvested crop.
```suggestion
		new SkillChallengeClue("Plant a watermelon seed.", item(ItemID.SEED_DIBBER), xOfItem(ItemID.WATERMELON_SEED, 3)),
```

Reply:
There's a surprising amount of people who don't have Auto Weed (because they don't like Tithe Farm), and those who do have it know they have it, so I think it's better to have the extra rake there.

Reply:
I'd argue most people would have easy access to a rake via a nearby tool leprechaun, so it seems extraneous to include a tool which is not directly needed, but that's my 2c :man_shrugging: